internal/cmd/kafka/command_topic_consume_onprem.go

```diff
@@ -52,7 +52,6 @@ func (c *authenticatedTopicCommand) newConsumeCommandOnPrem() *cobra.Command {
 	cmd.Flags().StringSlice("config", nil, `A comma-separated list of configuration overrides ("key=value") for the consumer client.`)
 	cmd.Flags().String("config-file", "", "The path to the configuration file (in json or avro format) for the consumer client.")
 	cmd.Flags().String("schema-registry-endpoint", "", "The URL of the Schema Registry cluster.")
-	pcmd.AddOutputFlag(cmd)
```

Comment:
This looks like a breaking change. Did you mean to delete this?

Reply:
(Also, is it even being used?)

Reply:
That's why it's being deleted... we never used this flag

Reply:
Can you add this to the "Bug Fixes" section of the release notes?

Reply:
will do!